src/Akka.Hosting/Logging/LoggerFactoryLogger.cs

```diff
@@ -0,0 +1,135 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="LoggerFactoryLogger.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Collections.Concurrent;
+using System.Linq;
+using System.Text;
+using Akka.Actor;
+using Akka.Dispatch;
+using Akka.Event;
+using Microsoft.Extensions.Logging;
+using LogLevel = Microsoft.Extensions.Logging.LogLevel;
+
+namespace Akka.Hosting.Logging
+{
+    public class LoggerFactoryLogger: ActorBase, IRequiresMessageQueue<ILoggerMessageQueueSemantics>
+    {
+        public const string DefaultTimeStampFormat = "yy/MM/dd-HH:mm:ss.ffff";
+        private const string DefaultMessageFormat = "[{{Timestamp:{0}}}][{{SourceContext}}][{{LogSource}}][{{ActorPath}}][{{Thread:0000}}]: {{Message}}";
```

Comment:
I'm not sure if this is the best way to format the string, I'm open to suggestions.